addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -2626,7 +2628,7 @@ export class OdooEditor extends EventTarget {
     _positionTableUi(element) {
         const isRow = element.nodeName === 'TR';
         const ui = isRow ? this._rowUi : this._columnUi;
-        const elementRect = element.getBoundingClientRect();
+        const elementRect = element && element.getBoundingClientRect();
```

Comment:
@visp-odoo in what case we wont find element?

Reply:
@shsa-odoo In Previous solution there are some points in which we could not found the element that the only reason why i wrote it here. But its not needed anymore.